spec/features/email_spec.rb

```diff
@@ -27,17 +27,30 @@
     expect(page).to have_text(I18n.t('.emails.submit.sending_successful'))
   end
 
-  scenario "logged in as Organizer after sending an Email to the applicants the event application status gets locked" do
+  scenario "logged in as Organizer after sending an acceptance Email to the applicants the event application status for sending acceptances gets locked" do
```

Comment:
Through the new model test you can ensure that if an acceptance mail has been sent the status gets locked. I think a test like this should only test a single functionality.